optional-kubernetes-engine/config/worker-supervisord.conf

```diff
@@ -1,11 +0,0 @@
-[program: pubsub-worker]
```

Comment:
Don't we want this file still for K8s Engine? It still uses the same docker image / runtime, and so the worker won't run without it.

Reply:
The GKE version is using the custom Dockerfile and it has `CMD` for running the worker.
Potentially we can move it to supervisord, but now I want to fix the step 6 first.